src/profiles/y_profile_admin_classes.prog.abap

```diff
@@ -558,6 +556,8 @@ CLASS lcl_util IMPLEMENTATION.
                                             header_text = 'Apply on Productive Code'(050) ).
         checks_tree->set_field_header_text( fieldname   = 'APPLY_ON_TESTCODE'
                                             header_text = 'Apply on Testcode'(034) ).
+        checks_tree->set_field_header_text( fieldname   = 'IGNORE_PSEUDO_COMMENTS'
+                                            header_text = 'Allow Pseudo Comments' ).
```

Comment:
Let's keep it in the database as 'ignore pseudo comments', and 'allow pseudo comments' in the screen. So, we avoid side effects in the already existing profiles.